subsys/fs/Kconfig

```diff
@@ -15,18 +15,18 @@ config FILE_SYSTEM
 	help
 	  Enables support for file system.
 
-config SYS_LOG_FS_LEVEL
+config LOG_FS_LEVEL
```

Comment:
use template

Reply:
@nashif pls check 7d70dc38c24c87f75996876adfa320ec9a4c7a59 for template changes